debug/org.eclipse.cdt.debug.ui/plugin.properties

```diff
@@ -282,3 +282,10 @@ popup.stepIntoSelection.name=Step Into Selection
 DebuggerConsoleView.name=Debugger Console
 
 GenericTarget.name=Generic Target
+
+# Toggle Instruction Step Mode Command Activity support
+org.eclipse.cdt.debug.ui.category.name=C/C++ Debug UI
+org.eclipse.cdt.debug.ui.category.description=Controls visibility of C/C++ Debug UI contributions
+
+org.eclipse.cdt.debug.ui.activity.toggleInstructionStepMode.name=Toggle Instruction Step Mode Command
```

Comment:
I believe it should be short and look like `Instruction Step Mode Button on main Toolbar`